ZenPacks/zenoss/Microsoft/Windows/modeler/plugins/zenoss/winrm/WinMSSQL.py

```diff
@@ -293,88 +294,56 @@ def __init__(self, conn_info, log):
         "  $ag_info['id'] = $ag_uid;"
         "  $ag_info['ag_res_id'] = $ag_res_uid;"
         "  $ag_info['primary_replica_server_name'] = $pr_rep_servr_name;"
-        # Put databases on top beacause we get them regardless of Replica role (either on Primary or Secondary Replicas)
-        "  foreach ($adb in $ag.AvailabilityDatabases) {"
-        "   $adb_inf = New-Object 'System.Collections.Generic.Dictionary[string, object]';"
-        "   $adb_inf['adb_id'] = $adb.UniqueId;"
-        "   $adb_inf['ag_res_id'] = $ag_res_uid;"
-        "   $adb_inf['name'] = $adb.Name;"
-        "   $adb_inf['sync_state'] = $adb.SynchronizationState;"
-        "   $adb_inf['suspended'] = $adb.IsSuspended;"
-        "   $db = $dbs | Where-Object {$_.AvailabilityGroupName -EQ $ag.Name -and $_.Name -EQ $adb.Name};"
-        "   $adb_inf['version'] = $db.Version;"
-        "   $adb_inf['isaccessible'] = $db.IsAccessible;"
-        "   $adb_inf['db_id'] = $db.ID;"
-        "   $adb_inf['owner'] = $db.Owner;"
-        "   $adb_inf['lastbackupdate'] = (New-TimeSpan -Start $epoch -End $db.LastBackupDate).TotalSeconds;"
-        "   $adb_inf['collation'] = $db.Collation;"
-        "   $adb_inf['createdate'] = (New-TimeSpan -Start $epoch -End $db.CreateDate).TotalSeconds;"
-        "   $adb_inf['defaultfilegroup'] = $db.DefaultFileGroup;"
-        "   $adb_inf['primaryfilepath'] = $db.PrimaryFilePath;"
-        "   $adb_inf['lastlogbackupdate'] = (New-TimeSpan -Start $epoch -End $db.LastLogBackupDate).TotalSeconds;"
-        "   $adb_inf['systemobject'] = $db.IsSystemObject;"
-        "   $adb_inf['recoverymodel'] = $db.DatabaseOptions.RecoveryModel;"
-        "   $adbs.Add($adb_inf) > $null;"
-        "  }"
-        "  if ($pr_rep_servr_name -ne $server.Name) {"
-        "   $ags.Add($ag_info) > $null;"
-        "   continue;"
-        "  }"
         "  $ag_info['is_distributed'] = $ag.IsDistributedAvailabilityGroup;"
         "  $ag_info['health_check_timeout'] = $ag.HealthCheckTimeout;"
         "  $ag_info['automated_backup_preference'] = $ag.AutomatedBackupPreference;"
         "  $ag_info['failure_condition_level'] = $ag.FailureConditionLevel;"
         "  $ag_info['cluster_type'] = $ag.ClusterTypeWithDefault;"
         "  $ag_info['db_level_health_detection'] = $ag.DatabaseHealthTrigger;"
-        "  $ag_uid = $ag.UniqueId;"
```

Comment:
Remove modeling of some Always On properties which then obtain during monitoring.